cmd/promtool/main.go

```diff
@@ -241,6 +241,13 @@ func main() {
 	dumpMaxTime := tsdbDumpCmd.Flag("max-time", "Maximum timestamp to dump.").Default(strconv.FormatInt(math.MaxInt64, 10)).Int64()
 	dumpMatch := tsdbDumpCmd.Flag("match", "Series selector. Can be specified multiple times.").Default("{__name__=~'(?s:.*)'}").Strings()
 
+	tsdbDumpSeriesCmd := tsdbCmd.Command("dump-series", "Dump duplicate series (remove samples content) from a TSDB.")
```

Comment:
"duplicate" is a typo?

Reply:
Why "remove samples content"?  Why would someone reading this think dump-series has anything to do with samples?

Reply:
Documentation should perhaps state the output is in JSON format.

Reply:
The doc has been updated to `Dump series (identified by a unique set of labels) from a TSDB into JSON format.` and now feels more readable